indra/newview/llplacesinventorypanel.cpp

```diff
@@ -120,3 +120,13 @@ S32	LLPlacesInventoryPanel::notify(const LLSD& info)
 	}
 	return 0;
 }
+
+BOOL LLPlacesInventoryPanel::handleDragAndDrop(S32 x, S32 y, MASK mask, BOOL drop, EDragAndDropType cargo_type, void *cargo_data,
+                                                      EAcceptance *accept, std::string &tooltip_msg)
+{
+    if (mAcceptsDragAndDrop) 
```

Comment:
mAcceptsDragAndDrop is declared in LLInventoryPanel, shouldn't it be checked there?

Reply:
I intentionally wanted these changes only for Places floater.